resources/metadata/cloudFoundryDeploy.yaml

```diff
@@ -388,3 +388,9 @@ spec:
               - name: cfApiEndpoint
               - name: cfOrg
               - name: cfSpace
+                outputs:
+      - name: commonPipelineEnvironment
+        type: piperEnvironment
+        params:
+          - name: cfGetAppUrl
```

Comment:
please prefix parameter name with `custom/` otherwise it cannot be passed properly via groovy layer.
Example:
https://github.com/SAP/jenkins-library/blob/master/resources/metadata/whitesource.yaml#L461